lib/utilities/aclUtils.js

```diff
@@ -233,4 +234,72 @@ aclUtils.sortHeaderGrants =
         return addACLParams;
     };
 
+/**
+ * convertToXml - Converts the `grantInfo` object (defined in `objectGetACL()`)
+ * to an XML DOM string
+ * @param {object} grantInfo - The `grantInfo` object defined in
+ * `objectGetACL()`
+ * @return {string} xml.join('') - The XML DOM string
+ */
+aclUtils.convertToXml = grantInfo => {
+    const { grants, ownerInfo } = grantInfo;
+    const xml = [];
+
+    // Escape any special XML characters
+    for (const key in grants) {
```

Comment:
You should use `Object.keys(grants).forEach` over `for(var x in )..` for consistency and potential performance issues https://github.com/petkaantonov/bluebird/wiki/Optimization-killers#5-for-in